gift-certificates-app/dao/src/test/java/com/epam/esm/dao/impl/OrderDAOImplTest.java

```diff
@@ -0,0 +1,69 @@
+package com.epam.esm.dao.impl;
+
+import com.epam.esm.bean.Certificate;
+import com.epam.esm.bean.Order;
+import com.epam.esm.config.ConfigDAO;
+import com.epam.esm.dao.CertificateDAO;
+import com.epam.esm.dao.OrderDAO;
+import com.epam.esm.dao.UserDAO;
+import com.epam.esm.exception.ResourceNotFoundException;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.jdbc.Sql;
+import org.springframework.transaction.annotation.Transactional;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@SpringBootTest(classes = {ConfigDAO.class})
+@TestPropertySource(
+        locations = "classpath:properties/application-test.properties")
+@Sql(scripts = "classpath:db_data.sql", executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD)
+@Sql(scripts = "classpath:drop.sql", executionPhase = Sql.ExecutionPhase.AFTER_TEST_METHOD)
+@Transactional
```

Comment:
do you need to have @Transactional here if you have @Transactional in dao classes?

Reply:
+ I have deleted @Transactional from @Repository